sonarqube/resource_sonarqube_webhook.go

```diff
@@ -52,6 +53,11 @@ func resourceSonarqubeWebhook() *schema.Resource {
 				Optional:  true,
 				Computed:  true,
 			},
+			"project": {
+				Type:        schema.TypeString,
+				Description: "The key of the project that will own the webhook (optional).",
```

Comment:
Nit: Can you remove t he `(optional` text from the Description. The line below already notes this field as optional